catalogue/webapp/components/WorkDetails/WorkDetails.js

```diff
@@ -380,26 +380,20 @@ const WorkDetails = ({
         </WorkDetailsSection>
       )}
       {!digitalLocation && (locationOfWork || encoreLink) && <WhereToFindIt />}
-      <TogglesContext.Consumer>
-        {({ newImageSearch }) =>
-          newImageSearch &&
-          work.images &&
-          work.images.length > 0 && (
-            <WorkDetailsSection headingText="Selected images from this work">
-              <ButtonOutlinedLink
-                text={`View ${work.images.length} images`}
-                link={worksLink(
-                  {
-                    search: 'images',
-                    query: work.id,
-                  },
-                  'work_details/images'
-                )}
-              />
-            </WorkDetailsSection>
-          )
-        }
-      </TogglesContext.Consumer>
+      {work.images && work.images.length > 0 && (
+        <WorkDetailsSection headingText="Selected images from this work">
+          <ButtonOutlinedLink
+            text={`View ${work.images.length} images`}
```

Comment:
Do we mind that this could be, 'View 1 images'?

Reply:
Ooh, something weird has happened - I had in fact removed this _and_ fixed the pluralisation in [`662bb68` (#5452)](https://github.com/wellcomecollection/wellcomecollection.org/pull/5452/commits/662bb68cf0c97825006cebb0f81a19fbf2dce32b)